frontend/src/hooks/Http/useHttp.ts

```diff
@@ -1,12 +1,18 @@
-import { useCallback, useMemo, useState } from 'react';
+import { useCallback, useContext, useMemo, useState } from 'react';
 import { ErrorResponse } from './ErrorResponse';
 import { HttpState } from './HttpState';
 import { HttpOptions } from './HttpOptions';
+import { UserContext } from '../../contexts/User/UserContext';
+import { User } from '../../contexts/User/User';
+
+const HTTP_STATUS_NO_CONTENT = 204;
```

Comment:
You can create an enum with all necessary HTTP statuses because these statuses can be used in multiple places.